corehq/apps/reports/static/reports/js/filters/select2s.js

```diff
@@ -100,6 +100,7 @@ hqDefine("reports/js/filters/select2s", [
             ajax: {
                 url: data.endpoint,
                 dataType: 'json',
+                delay: 250,
```

Comment:
nit: we have examples of both 250 & 500 in the code. But i would say 500.

Reply:
```suggestion
                delay: 500,
```